BackendServices/AuxiliaryServices/WebUtils/OHS/Statistic.cs

```diff
@@ -45,5 +45,46 @@ public class Statistic
             else
                 return JaminProcessor.JaminFormat("{ [\"status\"] = \"fail\" }", 0);
         }
+
+
+        public static string? Tracker(byte[] PostData, string ContentType)
+        {
+            string? dataforohs = null;
+
+            string? boundary = HTTPUtils.ExtractBoundary(ContentType);
+
+            if (boundary != null)
```

Comment:
It's best to check for !string.IsNullOrEmpty here

Reply:
I get your point here i can update for these

Reply:
Updated this check across the board of WebUtils. Marking as resolved.